packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile

```diff
@@ -18,14 +18,21 @@ RUN yum update -y \
 COPY requirements.txt ./
 RUN python -m pip install -r requirements.txt -t /opt/awscli
 
+#
```

Comment:
Can you remove the added empty lines?

Reply:
I added these to align with the style of the rest of the file. I can remove them everywhere if preferred.

Reply:
Ah in that case, it's fine with me! Thanks for explaining.